src/content/language/constructors.md

```diff
@@ -10,88 +10,68 @@ nextpage:
+Constructors create instances of classes.
 
-Declare a constructor by creating a function with the same name as its
-class (plus, optionally, an additional identifier as described in
-[Named constructors](#named-constructors)). 
+* To declare a constructor, create a function named the same as its class.
+  You can add an optional additional identifier as described in
+  [Named constructors](#named-constructors).
 
-Use the most common constructor, the generative constructor, to create a new
-instance of a class, and [initializing formal parameters](#initializing-formal-parameters)
-to instantiate any instance variables, if necessary:
+* To instantiate a class, use the
+  [generative constructor](#generative-constructors).
 
-<?code-excerpt "misc/lib/language_tour/classes/point_alt.dart (idiomatic-constructor)" plaster="none"?>
-```dart
-class Point {
-  double x = 0;
-  double y = 0;
+* To instantiate any instance variables,
+  [initialize formal parameters](#parameter-initialization).
 
-  // Generative constructor with initializing formal parameters:
-  Point(this.x, this.y);
-}
-```
+<?code-excerpt path-base="misc/lib/language_tour/classes"?>
 
-The `this` keyword refers to the current instance.
+<?code-excerpt replace="/ *\/\/\s+ignore_for_file:[^\n]+\n//g; /(^|\n) *\/\/\s+ignore:[^\n]+\n/$1/g; /(\n[^\n]+) *\/\/\s+ignore:[^\n]+\n/$1\n/g; / *\/\/\s+ignore:[^\n]+//g; /([A-Z]\w*)\d\b/$1/g"?>
 
-:::note
-Use `this` only when there is a name conflict. 
-Otherwise, Dart style omits the `this`.
-:::
+## Types of constructors
+
+### Constructor inheritance
+
+_Subclasses_, or child classes, don't inherit *constructors*
+from their _superclass_, or immediate parent class.
+If a class doesn't declare a constructor, it can only use the
+[default constructor](#default-constructors).
 
+A class can inherit the _parameters_ of a superclass.
+These are called [super parameters](#super-parameters)
+A subclass without constructor declarations can only use
+a [default constructor](#default-constructors).
 
-## Initializing formal parameters
+Constructors work in a somewhat similar way to
+how you call a chain of static methods.
+Each subclass can call its superclass's constructor to initialize an instance,
+like a subclass can call a superclass's static method.
+This process doesn't involve any "inheritance".
 
-Dart has *initializing formal parameters* to simplify the common pattern of
-assigning a constructor argument to an instance variable. 
-Use `this.propertyName` directly in the constructor declaration,
-and omit the body. 
+### Default constructors
 
-Initializing parameters also allow you to initialize
-non-nullable or `final` instance variables,
-which both must be initialized or provided a default value:
+If you don't declare a constructor, Dart uses the default constructor.
+The default constructor has no arguments and no name.
```

Comment:
[optional] I don't know for sure if the detail of "generative constructor" is specifically useful to understand the concept of the default constructor - but knowing that the default constructor is a generative constructor helps to understand the concept of generative constructors in my opinion. I think since this flows into the next section, it could be worth adding this detail. What do you think?

```suggestion
The default constructor is a generative constructor with no arguments and no name.
```